examples/chat_app.md

```diff
@@ -0,0 +1,251 @@
+# Building a chat app with Deno
+
+## Introduction
+
+In this tutorial we'll create a simple chat app using Deno. Our chat app will
+allow multiple chat clients connected to the same backend to send group messages
+through web sockets. After a client chooses a username, they can then start
+sending group messages to other online clients. Each client also displays the
+list of currently active users.
+
+![Untitled](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/4002626d-edc6-41dd-a518-c6197df4f450/Untitled.png)
+
+## Building the View
+
+We can build the simple UI shown above with the following as our `index.html`.
+Note that:
+
+1. We're using Bootstrap for styling
+2. The `app.js` script is our chat client (which will be discussed in detail
+   later)
+
+```html
+<!-- index.html -->
+
+<html>
+  <head>
+    <title>Node Chat App</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```

Comment:
I really really think we should avoid using jquery and bootstrap in an example application... it isn't modern.

Reply:
the updated code shouldn't have any jquery in it